geofront/server.py

```diff
@@ -959,6 +969,35 @@ def list_remotes(token_id: str):
     )
 
 
+@app.route('/tokens/<token_id:token_id>/remotes/<alias>/', methods=['GET'])
+def remote_info(token_id: str, alias: str):
+    """Retrieve the information of the given remote alias.
+
+    :param token_id: the token id that holds the identity
+    :type token_id: :class:`str`
+    :param alias: the alias of the remote to access
+    :type alias: :class:`str`
+    :status 200: when successfully granted a temporary authorization
+    :status 404: (``not-found``) when there's no such remote
+
+    """
```

Comment:
[`.. versionadded:: 0.5.0` directive][1] is missing.

[1]: http://www.sphinx-doc.org/en/stable/markup/para.html#directive-versionadded